src/main/groovy/de/jansauer/printcoverage/PrintCoveragePlugin.groovy

```diff
@@ -15,9 +15,15 @@ class PrintCoveragePlugin implements Plugin<Project> {
     }
 
     def extension = target.extensions.create('printcoverage', PrintCoverageExtension, target)
+
     Task task = target.tasks.create('printCoverage', PrintCoverageTask) {
       coverageType = extension.coverageType
+      reportFile = extension.reportFile
+      message = extension.message
     }
+
+    target.ext.PrintCoverage = PrintCoverageTask
```

Comment:
Allows to reference the PrintCoverageTask in build-scripts without having to use the fully qualified class name:
`task customTask(type: PrintCoverage) { ... }`
instead of
`task customTask(type: de.jansauer.printcoverage.PrintCoverageTask) { ... }`